packages/esm-ward-app/src/ward-patient-card/row-elements/ward-patient-time-on-ward.tsx

```diff
@@ -0,0 +1,16 @@
+import { age } from '@openmrs/esm-framework';
+import React from 'react';
+import { type WardPatientCardElement } from '../../types';
+import { useTranslation } from 'react-i18next';
+
+const WardPatientTimeOnWard: WardPatientCardElement = ({ encounterAssigningToCurrentInpatientLocation }) => {
+  const { t } = useTranslation();
+  if (encounterAssigningToCurrentInpatientLocation) {
+    const timeOnWard = age(encounterAssigningToCurrentInpatientLocation.encounterDatetime);
```

Comment:
oh, cool, does the age function handle what unit to display (minutes, hours, days, years?) based on the age?

Reply:
Yeah, when you and @ibacher were going back and forth about naming this function, I was going to weigh in that we might not want to use age-related terms, since the idea is to use it for things beyond age, not sure if we need to re-litigate that, but happy to see this used here.

Reply:
Yes! [Recent change](https://github.com/openmrs/openmrs-esm-core/pull/1100).

Reply:
Well, the specific cut-off points may or may not make sense outside of displaying a patient age, which is why I thought it made sense to keep the age-specific function about age. For instance, age differences less than an hour are displayed in minutes, but over an hour in hours, whereas here it may be more appropriate to have, e.g., "6 hours, 20 minutes".

Basically, as an EHR where patient ages are important, it makes sense to have a first-class function for formatting ages with clinically reasonable defaults. We likely should also have a first-class function for formatting durations, just likely with different cut-off points (and maybe more options).